Modules/gcmodule.c

```diff
@@ -31,7 +31,7 @@
 #include "pycore_pyerrors.h"
 #include "pycore_pystate.h"     // _PyThreadState_GET()
 #include "pydtrace.h"
-#include "pytime.h"             // _PyTime_GetMonotonicClock()
```

Comment:
Hum. Since it's included by Python.h, including it explicitly is not needed, no?

Reply:
I've updated these, and moved the `pytime.h` include further down in Python.h to allow for it. I believe there aren't any other files including `pytime.h` directly now.